frontend/src/components/EventCard/EventCard.tsx

```diff
@@ -0,0 +1,66 @@
+import { Card, CardHeader, CardContent, CardFooter } from "@/components/ui/card";
+import { Button } from "@/components/ui/button";
+import { Badge } from "@/components/ui/badge";
+
+export default function EventCard({ event }: any) {
+  const eventModality = event.modality === "in_person" ? "In Person" : "Online";
```

Comment:
Use the constants that you already created to get the label, is better to have only one source of truth